src/dioptra/restapi/v1/groups/schema.py

```diff
@@ -19,14 +19,15 @@
 
 from marshmallow import Schema, fields
 
+from dioptra.restapi.v1.users.schema import UserRefSchema
+
 
 class GroupRefSchema(Schema):
     """The reference schema for the data stored in a Group resource."""
 
     id = fields.Integer(
         attribute="id",
         metadata=dict(description="ID for the Group resource."),
-        dump_only=True,
```

Comment:
Should `dump_only` be removed here? I think it's true for every field in every ref schema, as all of those values are coming from, created, and modified on different objects. It's marked as such in the [base resource](https://github.com/usnistgov/dioptra/blob/d1303030f8a9435fb325d7765d77476fc4291bec/src/dioptra/restapi/v1/schemas.py#L35) but not in the [base ref](https://github.com/usnistgov/dioptra/blob/d1303030f8a9435fb325d7765d77476fc4291bec/src/dioptra/restapi/v1/schemas.py#L106). Is there a misunderstanding on my part @keithmanville ?

Reply:
> Should `dump_only` be removed here? I think it's true for every field in every ref schema, as all of those values are coming from, created, and modified on different objects. It's marked as such in the [base resource](https://github.com/usnistgov/dioptra/blob/d1303030f8a9435fb325d7765d77476fc4291bec/src/dioptra/restapi/v1/schemas.py#L35) but not in the [base ref](https://github.com/usnistgov/dioptra/blob/d1303030f8a9435fb325d7765d77476fc4291bec/src/dioptra/restapi/v1/schemas.py#L106). Is there a misunderstanding on my part @keithmanville ?

I think you're right none of the Ref Schema's should be marked as dump only.